readme.md

```diff
@@ -250,6 +309,24 @@ Default: `true`
 
 Set to `false` if you don't want to extend the environment variables when providing the `env` property.
 
+### options from child_process#spawn and child_process#exec
```

Comment:
Feels weird to have a heading here. It kinda makes it unclear how to use these options. I would rather have it like this:

```
---

Execa also accepts the below options which are passed directly to `child_process#spawn()`/`child_process#exec()`
---
```

Reply:
> `child_process#exec()`

We don't actually use `exec()` internally, so not sure why it's mentioned here.

Reply:
We don't use it, but we re-implement its options (`timeout`, `maxBuffer`, `killSignal`).

Reply:
Ok. I still think we should make that clearer, with something like a proposed above.

Reply:
Fixed in https://github.com/sindresorhus/execa/pull/256/commits/38c57c6f0a2718efd875d04cb04d25abf9353694